manifests/repo.pp

```diff
@@ -89,31 +101,34 @@
           $repo_path   = 'mongodb-org'
         }
 
-        $location = $facts['os']['name'] ? {
+        $_repo_location = $facts['os']['name'] ? {
           'Debian' => "https://${repo_domain}/apt/debian",
           'Ubuntu' => "https://${repo_domain}/apt/ubuntu",
           default  => undef
         }
+        $_keyring_location = "https://www.mongodb.org/static/pgp/server-${version}.asc"
```

Comment:
This seems to redirect to: https://pgp.mongodb.com/server-${version}.asc

Reply:
I copied that from the mongodb docs...
https://www.mongodb.com/docs/v5.0/tutorial/install-mongodb-on-ubuntu/#install-mongodb-community-edition

Isn't it best to keep it according to docs?